packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -763,20 +765,25 @@ export const useArbTokenBridge = (
   }
 
   async function withdrawTokenEstimateGas({
-    erc20L1Address,
     amount,
-    l2Signer
+    erc20L1Address
   }: {
-    erc20L1Address: string
     amount: BigNumber
-    l2Signer: Signer
+    erc20L1Address: string
   }) {
-    const estimatedL1Gas = await gasEstimator.erc20WithdrawalL1Gas(l2.provider)
+    const estimatedL1Gas = BigNumber.from(160000)
```

Comment:
We can write this as `BigNumber.from(160_000)` which makes it a bit easier to understand at a glance.

https://v8.dev/features/numeric-separators